unit_testing/unit_test.py

```diff
@@ -58,8 +58,8 @@
     test_climatology,
     test_wod_read_data,
     test_bgc_gridded_initialisation,
-    test_process_data_methods,
-    test_example_scripts,
+    # test_process_data_methods,
```

Comment:
@jpolton in future might be good to add a reason for the commenting out of this test and maybe include a ticket number for the work to fix them